request.proto

```diff
@@ -165,7 +165,7 @@ message NearestStopPointsRequest {
 
 message GraphicalIsochroneRequest {
     optional JourneysRequest journeys_request = 1;
-    required int32 min_duration               = 2;
+    repeated int32 duration                   = 4;
```

Comment:
I prefer step_duration, as :
https://api.navitia.io/.../isochrones?from=..&to=...&min_duration=0&step_duration[]=600&step_duration[]=1200&step_duration[]=2400&step_duration[]=3600&max_duration=7200

I think the URI is more blue like that.